content/sensu-go/5.15/release-notes.md

```diff
@@ -41,6 +44,81 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
 
 ---
 
+## 5.15.0 release notes
+
+**November 18, 2019** &mdash; The latest release of Sensu Go, version 5.15.0, is now available for download. This is our biggest release yet and includes an important update to our free entity limit. We added the federation cluster registration API as well as the APIKey resource, which you can use for API authentication. In addition, the 5.15.0 release includes support for sensu-backend environment variables and bug fixes that improve error logging for mutator execution and flap detection weighting for checks.
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.15.0.
+
+**IMPORTANT:**
+Sensu's free entity limit is now 100 entities. All [commercial features][79] are available for free in the packaged Sensu Go distribution up to an entity limit of 100. You will see a warning when you approach the 100-entity limit (at 75%).
+
+If your Sensu instance includes more than 100 entities, [contact us][90] to learn how to upgrade your installation and increase your limit. See [Discourse][91] for more information about our usage policy.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][79]) Added support for the [federation cluster registration API][85].
+- ([Commercial feature][79]) Added MSI and NuGet builds for [sensuctl][89]. Also, MSI and NuGet installations now add the bin directory to the system PATH on Windows.
+- ([Commercial feature][79]) Added HTTP DELETE access for the [license management API][92].
+- Added the [APIKey resource][86], with HTTP API support for POST, GET, and DELETE and [sensuctl commands][87] to manage the APIKey resource.
+- Added support for using [API keys for API authentication][88].
+- Added support for [sensuctl commands][93] to install, execute, list, and delete commands from Bonsai or a URL.
+- Added support for sensu-backend service environment variables.
+- Added support for [timezones in check `cron` strings][94].
```

Comment:
See https://github.com/sensu/sensu-go/commit/a93b17202ef29d7ffa29e212000aa148938ea541
```suggestion
- Added support for [timezones in check `cron` strings][94].
- Added support for extending sensuctl support with commands.
```

Reply:
@nikkixdev I have that listed above:
`Added support for [sensuctl commands][93] to install, execute, list, and delete commands from Bonsai or a URL.`